Makefile

```diff
@@ -68,7 +73,7 @@ dependencies:
 
 # release builds and installs release binaries.
 release: dependencies test-long
-	go install -a ./...
```

Comment:
wait WRONG!

REBUILD is a dependency, but then the binaries are built with test constants. REBUILD needs to be called again because the constants change between testing and releasing.